ht-4/index.js

```diff
@@ -0,0 +1,18 @@
+(() => {
+  const list = document.querySelectorAll('#tablist');
```

Comment:
use another selector. selector by id and querySelectorAll look strange in the same line.